src/ut/json_alarms_test.cpp

```diff
@@ -90,6 +90,46 @@ TEST_F(JSONAlarmsTest, DetailsTooLong)
   EXPECT_THAT(error, testing::MatchesRegex(".*'details' exceeds.*"));
 }
 
+TEST_F(JSONAlarmsTest, CauseTooLong)
```

Comment:
These tests look good, but I don't think they're sufficient. I think the UTs you've put in the snmp-handlers should go in here. Let's discuss tomorrow